test/mitmproxy/addons/test_proxyauth.py

```diff
@@ -1,8 +1,8 @@
 import binascii
-import ldap3
```

Comment:
If the tests don't depend on the `ldap3` package any more, please also remove it from the dep-list in `setup.py`.